src/main/java/core/basesyntax/shape/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.shape;
+
+import core.basesyntax.type.Color;
+
+public class Circle extends Shape {
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * this.radius * this.radius;
```

Comment:
Consider removing 'this' keyword when referring to 'radius' as it's not necessary.